### ISSUE TITLE
Make smooth scroll optional

### ISSUE THREAD

**Issue by bqv** [LithApp/Lith#112]
It's painfully slow and unreadable for a very long buffer list

**Comment by MartinBriza** [comment]
I don't think there's a reasonable way to turn it off but the performance can be improved.
How long is your buffer list? What do you mean by "unreadable"? 
What platform and hardware is this happening on please?

**Comment by bqv** [comment]
Linux, amd64 i7 max specs. The scrolling is painfully slow and hard to control, partly because during "scrolling" nothing is readable (can't read moving text). My buffer list is currently 108 long

**Comment by MartinBriza** [comment]
I regularly have around 50 buffers open on much weaker hardware and on Linux, too, so I don't see any reason for your performance to be worse. Is it possible you're not using correct graphics drivers? Maybe an issue with your Qt platform plugin?

As for reading scrolling text, I don't think removing motion from the items in the list will help with their comprehension. I suggest instead using the filter function that's built into the buffer list. It should also be controllable exclusively with keyboard.

**Comment by bqv** [comment]
I use wayland. Could you please at least make scrolling speed configurable then, if you want to continue using it as is?

**Comment by MartinBriza** [comment]
This is very likely a problem of setup, either on your box, or in your distro, it makes no sense for me to to hack around these issues, sorry.
To fix this, either check your configuration (e.g. doublecheck if you're not missing any relevant Qt platform or platform theme plugins) or get in touch with your Linux distribution package maintainers. 
Another option is the `qt6ct` package (hopefully shipped with your distro) that allows for fine-grained configuration of Qt applications that use it.
To sum it up, this is not caused and fixable by Lith, so I'll close this issue. However I'll try to help you with this so if you want to, continue posting into this thread or come get in touch on #lith on libera.chat.

**Comment by bqv** [comment]
My distro doesn't even package qt6, i'm using flatpak... It's fine, it's not worth my fighting with

**Comment by MartinBriza** [comment]
I'm sorry.
However, what distro and compositor are you using? Also what input device and driver (touchpad, trackpoint, mouse; libinput, synaptics,...) I will maybe check this out at some point.
